---
uint32 robot_status
